src/BenchmarksServer/Startup.cs

```diff
@@ -211,18 +210,126 @@ private static string CloneAndRestore(string path, ServerJob job)
 
             Debug.Assert(benchmarksDir != null);
 
-            // TODO: add project references to checked out repos
-            // https://github.com/aspnet/benchmarks/issues/185
+            var builtDirs = new List<string>();
+            var builtPackages = new List<BuiltPackage>();
 
-            // Restore in each dir
-            foreach (var dir in dirs)
+            foreach (var dir in dirs.Except(new[] { benchmarksDir }))
             {
+                UpdateNuGetConfig(path, dir, builtDirs);
+                UpdateDependencies(path, dir, builtPackages);
+
                 ProcessUtil.Run("dotnet", "restore", workingDirectory: Path.Combine(path, dir));
+                ProcessUtil.Run("cmd", "/c build.cmd build-pack", workingDirectory: Path.Combine(path, dir),
```

Comment:
Is this and the backslashes the only things we know of preventing this from working on mac and linux? Is that going to be fixed before merging?